### PR TITLE
Enable sorted itemsets on 2.37+

### PR DESCRIPTION
**Pull request by millerdev**
The other part of https://github.com/dimagi/Vellum/pull/906

@emord @orangejenny @ctsims

### CODE REVIEW

File: corehq/apps/app_manager/views/formdesigner.py [hunk 1]
```diff
@@ -117,6 +117,7 @@ def _form_too_large(app, form):
         'templated_intents': domain_has_privilege(domain, privileges.TEMPLATED_INTENTS),
         'custom_intents': domain_has_privilege(domain, privileges.CUSTOM_INTENTS),
         'rich_text': True,
+        'sorted_itemsets': app.build_version >= '2.37',
```
Comment (emord) [inline corehq/apps/app_manager/views/formdesigner.py]: Should go https://github.com/dimagi/commcare-hq/blob/45692eb201b822f85a8011656c9b2d79e28f336c/corehq/apps/app_manager/feature_support.py

Comment (millerdev) [inline corehq/apps/app_manager/views/formdesigner.py]: thanks. fbb7bc1e488c757ac945fc62786e0113f6c1c3d5

**Review by emord — APPROVED** [review]

File: corehq/apps/app_manager/feature_support.py [hunk 2]
```diff
@@ -135,3 +135,10 @@ def supports_practice_users(self):
         Ability to configure practice mobile workers for apps
         """
         return self._require_minimum_version('2.26')
+
+    @property
+    def enable_sorted_itemsets(self):
+        """
+        Enable sorted itemsets in the form builder.
+        """
+        return self._require_minimum_version('2.37')
```
Comment (ctsims) [inline corehq/apps/app_manager/feature_support.py]: This should actually be 2.38 based on a conversation with Aliza

**Comment by millerdev** [comment]
@ctsims will it be a problem if I deploy this today? Like will Nick N. and others using this feature be able to test their apps if the Sort Field is disabled for apps on build versions below 2.38?

**Comment by ctsims** [comment]
That will be fine, the feature won't be working for them anyway.

I've made sure that internal people can move their apps to 2.38.0 to keep
things editable.

-Clayton

**Pull request merged**